test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -2059,4 +2061,79 @@ public void otherSituationsEndCounter(String pref, String fixture) {
       }
     }
   }
+
+  @Nested
+  class SpookyVHSTape {
+    @Test
+    public void canRecogniseSpookyVHSTapeMonster() {
+      var cleanups =
+          new Cleanups(
+              withProperty("spookyVHSTapeMonster", "ghost"),
```

Comment:
also withoutCounters()?

Reply:
This one inherently tidies itself because if any of the three counters it makes are still active, the assertions will fail. Still, I'll move this to using `withCounter`